ui/apps/vibes-console/src/pages/applications.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { useRootComponentProps } from '@akashaorg/ui-awf-hooks';
+import { JoinVibesCard } from '../components/applications/join-vibes-card';
+import Stack from '@akashaorg/design-system-core/lib/components/Stack';
+
+export const Applications: React.FC<unknown> = () => {
+  const { t } = useTranslation('vibes-console');
+  const { getRoutingPlugin } = useRootComponentProps();
+
+  const navigateTo = getRoutingPlugin().navigateTo;
+
+  const isModerator = false;
+
+  const handleCtaButtonClick = () => {
+    navigateTo?.({
```

Comment:
if you are using tanstack-router in this app, then also switch to useNavigate from that library for internal navigation